docs/mirroring/README.md

```diff
@@ -22,6 +22,15 @@ See: https://kubernetes.io/docs/contribute/style/content-guide/#dual-sourced-con
 
 <!--TODO: Generically identifying registry.k8s.io images in manifests / charts / addons.-->
 
+If you have a running cluster then our [community-images] [krew] plugin can
+help you identify Kubernetes Project image references to mirror like this:
+
+```console
+kubectl community-images --mirror
```

Comment:
putting this tool inline because it's very very broadly useful to inventory images without much context on what the user is using. we should expand on this a bit.

Reply:
expanded a bit more in later commit